.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
```

Comment:
- [ ] Please make sure you set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository. Because there are should be 4 types of linters set up so 1 is missing which is eslint. Take a look at the screenshot below to see the problem and kindly fix it 👇 
![Screenshot (211)](https://user-images.githubusercontent.com/67087664/156152001-8203cec2-4212-449c-9750-467213c825e9.png)